all-spark-notebook/README.md

```diff
@@ -8,11 +8,12 @@
 * Jupyter Notebook 4.3.x
 * Conda Python 3.x and Python 2.7.x environments
 * Conda R 3.3.x environment
-* Scala 2.10.x
+* Scala 2.11.x
 * pyspark, pandas, matplotlib, scipy, seaborn, scikit-learn pre-installed for Python
 * ggplot2, rcurl preinstalled for R
 * Spark 2.0.2 with Hadoop 2.7 for use in local mode or to connect to a cluster of Spark workers
 * Mesos client 0.25 binary that can communicate with a Mesos master
+* spylon-kernel
```

Comment:
Maybe move this into line 12 with the other Python libs, even though it's a kernel.

Can you also link the text to the repo?